internal/services/account_service.go

```diff
@@ -0,0 +1,51 @@
+package services
+
+import (
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	"github.com/RHEnVision/provisioning-backend/internal/payloads"
+	"github.com/go-chi/render"
+	"net/http"
+)
+
+func ListAccounts(w http.ResponseWriter, r *http.Request) {
```

Comment:
Should we even expose Accounts in API? it seems unecessary as the service should always run in a context of one account IMHO, right?

Reply:
Yes I know, but I wanted to create few example API calls with DAO so we have at least something, I had to start with Accounts because of dependencies. We will either delete REST operations completely or create an administrative role. It would be good if you could figure out if administrative roles are allowed by SRE or is something that is a good practice.